tts/README.md

```diff
@@ -0,0 +1,18 @@
+# Parler TTS API Example
+
+A text-to-speech API service powered by Parler TTS
+
+## Deployment
+
+```bash
+beam deploy app.py:generate_speech
+```
+
+## API Usage
+Send a POST request with:
+```bash
+{
+    "prompt": "Your text to convert to speech",
```

Comment:
would be good to add an real output from the model here

Reply:
ideally a screenshot or whatever the asset generated from the model is

Reply:
Added example audio